src/Cargo.lock

```diff
@@ -255,16 +255,16 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "clap"
-version = "2.25.1"
+version = "2.26.0"
```

Comment:
Yes, because I cleaned up orphan dependencies.

I don't know any way to make Cargo regenerate the lock without bumping unrelated dependencies, like NPM.